fed_test_utils/fed.go

```diff
@@ -149,6 +149,7 @@ func NewFedTest(t *testing.T, originConfig string) (ft *FedTest) {
 	viper.Set("Registry.RequireCacheApproval", false)
 	viper.Set("Director.CacheSortMethod", "distance")
 	viper.Set("Director.DbLocation", filepath.Join(t.TempDir(), "director.sqlite"))
+	viper.Set("Server.DbLocation", filepath.Join(t.TempDir(), "server.sqlite"))
```

Comment:
Heads up, this will conflict with a small refactor I just merged. You should rebase on main, and once the database name is changed, this should end up being :
```
viper.Set(param.Cache_DbLocation.GetString(), filepath.Join(t.TempDir(), "server.sqlite"))`
```